ccd-definition/CaseEvent/Camunda/BusinessProcessEvents-CUI-nonprod.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
```

Comment:
For release2 we agreed to have R2 post fix so would be BusinessProcessEvents-R2-CUI-nonprod.json. so it does not interfere with release1 which is still in progress

Reply:
Updated as suggested